test/src/test/java/org/infinispan/jmx/annotations/MBean.java

```diff
@@ -1,14 +1,19 @@
 package org.infinispan.jmx.annotations;
 
+import java.lang.annotation.Documented;
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Inherited;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * This is a copy of the real annotation found in Infinispan source tree. Should be kept in sync.
```

Comment:
Do these annotations really need to be kept in sync with the ones in the main repo?

Reply:
Interesting question. Depends. You might get away with it sometimes. You never know until you run a test and it breaks. But we don't have tests. And the annotations elements are accessed in an unsafe way via Element.getAnnotationMirrors(), not via the type safe AnnotatedConstruct.getAnnotationsByType(Class<A> annotationType). But we do not have those annotation classes here because they are in another module. Adding a dependency would lead to circularity. So that explains why we use the mirrors. But then accessing the element ```objectName``` or ```description``` of annotation MBean might lead to an error when executing with the real annotations if they are not in sync.